blog-content/lw4-d1-open-telemetry.md

```diff
@@ -0,0 +1,38 @@
+---
+title: "Day 1: Simplify tracing with OpenTelemetry and Highlight"
+createdAt: 2024-01-30T12:00:00Z
+readingTime: 3
+authorFirstName: Chris
+authorLastName: Esplin
+authorTitle: Software Engineer
+authorTwitter: 'https://twitter.com/chrisesplin'
+authorLinkedIn: 'https://www.linkedin.com/in/epsilon/'
+authorGithub: 'https://github.com/deltaepsilon'
+authorWebsite: 'https://www.chrisesplin.com/'
+authorPFP: 'https://firebasestorage.googleapis.com/v0/b/quiver-pixels-2020.appspot.com/o/F1EQ3eaBqkbEKEHBigolXIlmdut2%2F1408a808-60a6-4102-b636-08ab24041503.jpeg?alt=media&token=5f0ed5d8-c192-4aa3-a75b-3eb6cac9a552'
+image: 'https://www.highlight.io/_next/image?url=https%3A%2F%2Fmedia.graphassets.com%2FE7U4wuSyS5mXKGfDOWsz&w=3840&q=75'
+tags: Launch Week 4, OpenTelemetry
+metaTitle: "Day 1: Simplify tracing with OpenTelemetry and Highlight"
+---
+
+## Highlight, powered by OpenTelemetry
+
+Instrumenting code for performance tracing is a grind.
+
+Even worse, many observability solutions create vendor lock-in. All of that tracing work will send your data to exactly one kind of server, that you likely do not control.
+
+## The Unified Solution: OpenTelemetry and Highlight
+
+We're excited to introduce a powerful and flexible solution combining OpenTelemetry (OTel) with Highlight. OpenTelemetry provides comprehensive tracing, logging, metrics, and error monitoring capabilities for your application without vendor lock-in. It's a solution that is both opinionated and adaptable in terms of data storage - you can stream data to ClickHouse, Prometheus, or other open-source databases using their processors and exporters.
+
+Highlight complements OpenTelemetry by significantly reducing the engineering overhead associated with storing and visualizing this data. Simply point your OpenTelemetry collector to Highlight - whether Cloud or Self-Hosted - and we take care of the rest. Using Kafka and ClickHouse, we ingest the data efficiently and provide you with a robust UI for searching and viewing your application's tracing data.
```

Comment:
after this paragraph would be a good place to link the blog post? https://www.highlight.io/docs/getting-started/native-opentelemetry/overview

Reply:
maybe should also link it at the end if someone reads the full post.

Reply:
Done!